parity-util-mem/Cargo.toml

```diff
@@ -20,12 +20,12 @@ wee_alloc = { version = "0.4.5", optional = true }
 mimallocator = { version = "0.1.3", features = ["secure"], optional = true }
 mimalloc-sys = { version = "0.1.6", optional = true }
 
-elastic-array = { version = "0.10.2", optional = true }
+smallvec = { version = "1.0.0", optional = true }
 ethereum-types = { version = "0.8.0", optional = true, path = "../ethereum-types" }
 parking_lot = { version = "0.9.0", optional = true }
 
 [target.'cfg(target_os = "windows")'.dependencies]
-winapi = "0.3.8"
+winapi = { version = "0.3.8", features = ["heapapi"] }
```

Comment:
so it didn't work before on windows?

Reply:
I honestly don't know how it could have worked; looks like the `winapi` went completely crazy with features at some point and didn't care much for semver. 
The risks of updating dependencies with `cargo upgrade`...